test/acceptance/test/add_tests.go

```diff
@@ -1075,15 +1077,17 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 
 		By("And I run wego app add command for app", func() {
-			runWegoAddCommand(repoAbsolutePath, addCommand, WEGO_DEFAULT_NAMESPACE)
+			output, _ := runWegoAddCommandWithOutput(repoAbsolutePath, addCommand, WEGO_DEFAULT_NAMESPACE)
+			re := regexp.MustCompile(`(http|ftp|https):\/\/([\w\-_]+(?:(?:\.[\w\-_]+)+))([\w\-\.,@?^=%&amp;:/~\+#]*[\w\-\@?^=%&amp;/~\+#])?`)
```

Comment:
What is this checking for? A valid URL? Should `ftp` be allowed?

Reply:
no it probably shouldnt.